src/main/java/QueryServlet.java

```diff
@@ -140,6 +159,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         && queryToDataRowPost2013.get(action).containsKey(personType)) {
       dataRow = queryToDataRowPost2013.get(action).get(personType);
     }
+    if (year == 2010
```

Comment:
if this is guaranteed to be every 10 years you can do modulo 10.

Reply:
I was thinking about that, but from my preliminary poking around it seems like that rule will not necessarily hold fast - the only other modulo 10 years accessible by API are 1990 and 2000, and unfortunately it looks like 1990 has different mappings. So I left it this way for now, to be changed once I start working on getting 1990 and 2000 in.